params/config.go

```diff
@@ -317,7 +317,7 @@ var (
 		ArrowGlacierBlock:   nil,
 		ArchimedesBlock:     big.NewInt(0),
 		ShanghaiBlock:       big.NewInt(0),
-		BernoulliBlock:      nil,
+		BernoulliBlock:      big.NewInt(5016616),
```

Comment:
I think this block number is unconfirmed.

Reply:
When enable sepolia Bernoulli, need update this block number, i think

Reply:
Let's set Sepolia upgrade time to be 4/15 (Mon) morning China time, as Peter and most infra members will be in China time zone.

Reply:
by the way how to you estimate block number? Sepolia's block time isn't a fix number because of the lack of transactions (https://sepolia.scrollscan.com/).

Reply:
Updated. Rationale:

```
(3747132 - 3501781) * 3 + 1712410359 = 1713146412
Mon Apr 15 2024 10:00:12 GMT+0800 (China Standard Time)
```

>  Sepolia's block time isn't a fix number because of the lack of transactions

Yes, that's a very good point and I forgot about this. I started a script on `sepolia-l2geth-rpc0` (see `~/peter/send-txs.sh`), this way we should be able to ensure the 3s block time.